ios/EnrichedTextInputView.mm

```diff
@@ -218,6 +219,31 @@ - (void)setupPlaceholderLabel {
   _placeholderLabel.hidden = YES;
 }
 
+- (void)mediaAttachmentDidUpdate:(NSTextAttachment *)attachment {
```

Comment:
I will remove this one when https://github.com/software-mansion/react-native-enriched/pull/318 will be merged